lib/runner/wd-instances/base-wd-server.js

```diff
@@ -90,6 +90,9 @@ class BaseWDServer {
     this.pollStarted = false;
     this.processCreatedTimeout = null;
     this.processStatusCheckTimeout = null;
+    // for multi docker container increase max status polling retries and intervals.
+    this.maxStatusPollTries = settings.max_status_poll_tries || BaseWDServer.maxStatusPollTries();
```

Comment:
`BaseWDServer.maxStatusPollTries` is a static property, not a method and also I don't see `maxStatusPollTries` being converted to an instance property, it is still static: https://github.com/taitelman/nightwatch/blob/fixDriverPolling/lib/runner/wd-instances/base-wd-server.js#L60

Reply:
correct. fixing...
maxStatusPollTries is converted to instance property in line 94

Reply:
> I don't see any commits from #2144 and also there are some issues here so I'll go ahead and merge the initial PR.

indeed. I did it manually since I changed few items at once.